board/safety/safety_ford.h

```diff
@@ -249,8 +296,7 @@ static int ford_fwd_hook(int bus_num, int addr) {
 }
 
 static const addr_checks* ford_init(uint16_t param) {
-  UNUSED(param);
-
+  ford_longitudinal = GET_FLAG(param, FLAG_FORD_LONG_CONTROL);
```

Comment:
```suggestion
#ifdef ALLOW_DEBUG
  ford_longitudinal = GET_FLAG(param, FLAG_FORD_LONG_CONTROL);
#else
  hyundai_longitudinal = false;
#endif
```

we put this behind the debug flag so you can only enable on non-release panda